Dockerfile

```
@@ -73,15 +69,7 @@ ENV PATH="/opt/miniconda3/bin:${PATH}"
 
 # Install Miniconda on x86 or ARM platforms
 RUN arch=$(uname -m) && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)